homeassistant/components/freebox/router.py

```diff
@@ -161,9 +162,16 @@ async def _update_raids_sensors(self) -> None:
         """Update Freebox raids."""
         # None at first request
         try:
-            fbx_raids: list[dict[str, Any]] = await self._api.storage.get_raids() or []
+            fbx_raids: list[dict[str, Any]] = (
+                []
+                if not self.supports_raid
```

Comment:
Please don't use a ternary operator expression that spans more than one line. Also please only wrap the line that can raise inside the try... except block. The check should move up out of the try... except. Remove the `else:` block and outdent the code in that block to always run.